### DISCUSSION TITLE
Scrollbar when zooming

### DISCUSSION THREAD

**Discussion by sojastar** [katspaugh/wavesurfer.js#2775] [Q&A]
Hello,

Are there any options for the scrollbar that appears when the waveform is wider than its container? The zooming example shows a scrollbar under the waveform when it overflows. In my app, a tiny scrollbar appears under the waveform when it overflows but is soon fades away. I have tried to access it and style it with CSS, but I don't see the HTML element (if there is one).

Here is how I create my wavesurfer instance:
```javascript
let noteEditorWavesurfer = WaveSurfer.create({
  container:      document.querySelector("#noteEditorWaveform"),
  barWidth:       9,
  barHeight:      1,
  barGap:         9,
  waveColor:      "black",
  progressColor:  "white",
  interact:       false,
  scrollParent:   true,
  fillParent:     false,
  plugins: [
    WaveSurfer.regions.create({
      regionsMinLength: 0.1,
      regions:          [ { id:     "noteEditorRegion",
                            start:  note.in,
                            end:    note.out,
                            loop:   false,
                            drag:   false,
                            color: 'rgba(1.0, 1.0, 1.0, 0.2)' } ] } ) ]
});
```
I have tried to change `interact` to `true`, but that hasn't changed anything.

Thank you very much!

**Comment by katspaugh** [comment]
So you want the scrollbar to be always visible?

**Reply by sojastar** [reply]
Yes.

**Reply by katspaugh** [reply]
AFAIK, on windows it's always visible by default (as long as there's something to scroll).
On mac, you can override the default system settings with some CSS for `#noteEditorWaveform > wave`: https://codepen.io/ppolyzos/pen/AWKgOv.

**Reply by sojastar** [reply]
Sorry. I was testing inside the Firefox mobile emulator and it looks like iOS always hides the scrollbar. It's properly there when I switch to desktop mode.

Thanks for the pointer though! And for your always so prompt replies!

**Comment by sojastar** [comment]
Ooops, closing the discussion.
